File: Pset6/sentimental/Mario/mario.py
from cs50 import get_int

height = -1
while True:
    altura = get_int("Heinght: ")
    if altura >= 0 and altura < 24:
        break

for fila in range(1, altura+1, 1):
    for columna in range(1, (altura - fila + 1), 1):
        print(" ", end="")
    for columna in range(0, fila + 1, 1):
        print("#", end="")
    print()

File: Pset1/Mario/mario.c
#include <stdio.h>
#include <cs50.h>
int main(void)
{
    int height;

    do
    {
        height =  get_int("Height: ");

    }
    while (height > 23 || height < 0);

    for (int cont1 = 1 ; cont1 <= height ; cont1++)
    {
        for (int cont2 = 0 ; cont2 <= height ; cont2++)
        {
            if (cont1 + cont2 <= height - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }

        }
        printf("\n");
    }



}